deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_crd.yaml

```diff
@@ -157,6 +157,13 @@ spec:
                   targetName:
                     description: Optional target name for registration
                     type: string
+              logging:
+                 description: Optional logging level configuration. Changing this value does not require restart of Kubeturbo
+                 type: object
+                 properties:
+                   level:
+                     description: Define logging level, default is info = 2
+                     type: integer
```

Comment:
Should we add min and max here to bound the logging levels we support?

Reply:
@shayne-lin  Thanks for your review and suggestion.  We don't have a validation for the logging level in the command line argument in the CRD, if we want to do that taking this opportunity,  we need to decide the minimum and maximum logging level the user can specify.  I think we need the team's opinion on that.

Reply:
Right. As you pointed it out [here](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#what-method-to-use), the recommended levels are from 0-5. Currently, I am seeing glog.V(1) to [glog.V(6)](https://github.com/search?q=repo%3Aturbonomic%2Fkubeturbo+glog.V%286%29&type=code) in the code. Not sure if there's a mismatch or if it was intentional. Maybe someone can comment on this?

Reply:
Upon some manual testing, setting log level to 7 or above does not break the code. It will act as if it's set to the highest available logging level and print all the messages. Therefore, there is no need to enforce min and max in the CRD.

Reply:
@shayne-lin  Thanks for the testing and update